plugins/Widgetize/lang/en.json

```
@@ -4,6 +4,19 @@
         "PluginDescription": "Display any Matomo report in your website or app with a simple \"embed\" HTML tag.",
         "ViewAccessRequired": "This user has at least some write access. Only tokens of users who have only view access can be used. Check %1$s for more info.",
         "TopLinkTooltip": "Export Matomo reports as widgets and embed the dashboard in your app as an iframe.",
-        "TooHighAccessLevel": "This user has super-user access. Embedding widgets with super-user token authentication is not allowed. %1$sCheck the FAQ for more info.%2$s"
+        "TooHighAccessLevel": "This user has super-user access. Embedding widgets with super-user token authentication is not allowed. %1$sCheck the FAQ for more info.%2$s",
+        "SelectAReport": "Select a report, and copy paste in your page the embed code below the widget:",
+        "Reports": "Widgetize reports",
+        "Intro": "With Matomo, you can %s on your blog, website, or intranet dashboard... in one click.",
+        "Intro2": "export your Web Analytics reports",
+        "DisplayDashboardInIframe": "You can also display the full Matomo dashboard in your application or website in an IFRAME (%s).The date parameter can be set to a specific calendar date, \"today\", or \"yesterday\". The\n period parameter can be set to \"day\", \"week\", \"month\", or\n \"year\". The language parameter can be set to the language code of a translation, such as language=fr. For example, for idSite=1 and date=yesterday, you can write: <br/>",
```

Might be better to move the `<br />` outside of the translation into the vue code. Otherwise it could be missing if a translator doesn't add it.
Also the `\n` can be simply removed as it doesn't have any effect here.
And a space is missing...

```suggestion
        "DisplayDashboardInIframe": "You can also display the full Matomo dashboard in your application or website in an IFRAME (%s). The date parameter can be set to a specific calendar date, \"today\", or \"yesterday\". The period parameter can be set to \"day\", \"week\", \"month\", or\n \"year\". The language parameter can be set to the language code of a translation, such as language=fr. For example, for idSite=1 and date=yesterday, you can write:",
```